game.py

```diff
@@ -1,62 +1,131 @@
-import random
+from random import shuffle
 
 
 class Card:
-    def __init__(self, value, suit):
-        self.value = value
+    def __init__(self, rank, suit):
+        self.rank = rank
         self.suit = suit
 
+    def check_card_value(self, card):
+        ACE = [1]
+        ROYALTY = [11, 12, 13]
+        rank = card.rank
+
+        if rank in ROYALTY:
+            return 10
+        elif rank in ACE:
+            return (1, 11)
+        else:
+            return rank
+
     def __str__(self):
-        return f"{self.value}{self.suit}"
+        return f"{self.rank}{self.suit}"
 
     def __repr__(self):
-        return f"Card('{self.value}', '{self.suit}')"
+        return f"Card('{self.rank}', '{self.suit}')"
 
 
 class Deck:
 
-    CARD_VALUE = ["2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
-    CARD_SUIT = ["\u2663", "\u2665", "\u2666", "\u2660"]
+    CARD_SUITS = ["♠️", "♣️", "♥️", "♦️"]
 
     def __init__(self):
-        self.deck = []
-        self.new_deck()
+        self.cards = [
+            Card(rank, suit) for suit in self.CARD_SUITS for rank in range(1, 14)
+        ]
+
+        self.cards = self.shuffle_deck(self.cards)
 
-    def new_deck(self):
-        for suit in self.CARD_SUIT:
-            for value in self.CARD_VALUE:
-                self.deck.append(Card(value, suit))
-        random.shuffle(self.deck)
+    def shuffle_deck(self, cards):
```

Comment:
more explicit, nice
do you need to pass in cards, you could use self.cards?
this might also be nice as a classmethod maybe?